backend/corpora/dataset_processing/process.py

```diff
@@ -459,7 +465,7 @@ def validate_h5ad_file_and_add_labels(dataset_id, local_filename):
             validation_status=ValidationStatus.VALID,
         )
         update_db(dataset_id, processing_status=status)
-        return output_filename
+        return dict(labeled_filename=output_filename, can_convert_to_seurat=can_convert_to_seurat)
```

Comment:
Personally, when the use case is fairly simply, I strongly prefer we use the implicit tuple return of `one, two, three` rather than a dictionary that we then have to use keys on later. Kind of like how `validate.validate` just returns the tuple on ln 450

Reply:
https://github.com/chanzuckerberg/single-cell-data-portal/pull/1654 @metakuni